src/Dofs/MixedDofHandler.jl

```diff
@@ -0,0 +1,254 @@
+
+mutable struct FieldHandler
+    field_names::Vector{Symbol}
```

Comment:
Maybe add a `struct Field` with`field_name`, `field_dims` and `and field_interpolations`

Reply:
I have done so now. The FieldHandler stores a number of Fields.